### PR TITLE
[chmlib] support on arm

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: ports/chmlib/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,14 @@
 {
   "name": "chmlib",
-  "version-string": "0.40",
-  "port-version": 5,
+  "version": "0.40",
+  "port-version": 6,
   "description": "CHMLIB is a library for dealing with Microsoft ITSS/CHM format files. Right now, it is a very simple library, but sufficient for dealing with all of the .chm files I've come across. Due to the fairly well-designed indexing built into this particular file format, even a small library is able to gain reasonably good performance indexing into ITSS archives.",
-  "homepage": "https://www.jedrea.com/chmlib/"
+  "homepage": "https://www.jedrea.com/chmlib/",
```
Comment (chausner) [inline ports/chmlib/vcpkg.json:6]:
```suggestion
  "homepage": "http://www.jedrea.com/chmlib/",
```
Site does not support HTTPS.

File: ports/chmlib/portfile.cmake [hunk 2]
```diff
@@ -13,21 +13,21 @@ vcpkg_download_distfile(
...
 
-file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt" DESTINATION "${SOURCE_PATH}")
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS_RELEASE -DBUILD_TOOLS=ON
```
Comment (vicroms) [inline ports/chmlib/portfile.cmake:25]: Is this actually installing any tools? I don't see any tool files being copied into the installed directory.

Comment (autoantwort) [reply ports/chmlib/portfile.cmake:25]: Yes: 
image

Comment (autoantwort) [reply ports/chmlib/portfile.cmake:25]: There are build here:
https://github.com/microsoft/vcpkg/blob/master/ports/chmlib/CMakeLists.txt#L6-L17

**Pull request merged**